packages/main/src/RPA/Smartsheet.py

```diff
@@ -0,0 +1,1415 @@
+        argument, they will be appended to the end of the table as
+        additional columns in the data model. These additional objects
+        will be attached as a list of objects depending on the items
+        requested.
+
+        .. warn:
+
+            Only the following supported additional data requests
+            are supported when unpacking a Sheet object.
+
+            * ``attachments``: The value for the cell will be a list of
+            native Attachment object.
+            * ``attachmentFiles``: The value for the cell will be a list of
+            file paths to the local copies of each attachment. This
+            operation may be resource intensive for many attachments.
+            * ``discussions``: The value for the cell will be a list of
+            native Discussion objects.
+            * ``rowPermalink``: The value for the cell will be a single
+            string representing the URL to the permalink.
+        """
+        if sheet is None:
+            self._require_current_sheet()
+            sheet = self.current_sheet
+        includes = getattr(sheet, "selected_includes", []) or []
+        table_data = []
+        headers = []
+        for column in sheet.columns:
+            headers.append(column.title)
+        if len(includes) > 0:
+            headers.extend(includes)
+
+        for row in sheet.rows:
+            row_data = []
+            for cell in row.cells:
+                row_data.append(cell.value)
+            for key in includes:
+                key_or_func = self.SHEET_INCLUDES[key]
+                if callable(key_or_func):
+                    row_data.append(key_or_func(row))
+                elif getattr(row, key_or_func, False):
+                    row_data.append(getattr(row, key_or_func, None))
+
+            table_data.append(row_data)
+
+        table = Table(table_data, columns=headers)
+
+        return table
+
+    @keyword
+    def refresh_sheet(self, native: bool = False) -> Union[TableType, Sheet]:
+        """Refreshes the current sheet from the API and returns it
+        either as a Table or native data model depending on the
+        ``native`` argument.
+        """
+        self._require_current_sheet()
+        sheet = self.smart.Sheets.get_sheet(self.current_sheet.id)
+        self.current_sheet = sheet
+        if native:
+            return sheet
+        else:
+            return self.convert_sheet_to_table(sheet)
+
+    @keyword
+    def get_sheet_owner(
+        self, sheet_id: int = None, sheet_name: int = None
+    ) -> Tuple[str, int]:
+        """Returns the owner's username and ID for the current sheet."""
+        sheet_id = self._parse_sheet_id(sheet_id, sheet_name)
+        sheet = self.smart.Sheets.get_sheet(sheet_id, include=["ownerInfo"])
+        return sheet.owner, sheet.owner_id
+
+    @keyword
+    def list_sheet_filters(
+        self, sheet_id: int = None, sheet_name: int = None
+    ) -> List[SheetFilter]:
```

Comment:
```suggestion
    ) -> Optional[List[SheetFilter]]:
```
and everywhere else you return with `self._unpack_index_result(...)`.

Reply:
got it